requirements.txt

```diff
@@ -10,6 +10,7 @@ Pillow
 torch==1.2.0
 torchvision==0.4.0
 tensorboardX==1.8
+Click<7.2.0
```

Comment:
Force the version for Click. Otherwise, Click==8.0 gets installed and is incompatible with some dependencies of ai2thor.